com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/StagingResolver.java

```diff
@@ -0,0 +1,62 @@
+package com.sap.cloud.lm.sl.cf.process.util;
+
+import java.util.UUID;
+
+import org.cloudfoundry.client.lib.CloudControllerClient;
+import org.cloudfoundry.client.lib.domain.CloudBuild;
+import org.cloudfoundry.client.lib.domain.PackageState;
+
+import com.sap.cloud.lm.sl.cf.process.Constants;
+import com.sap.cloud.lm.sl.cf.process.steps.ExecutionWrapper;
+
+public class StagingResolver {
```

Comment:
Now that I think about it, `Resolver` does not express the purpose of the class very clearly. Consider using something like `ApplicationStager`.